terraform-ci/pipeline.yaml

```diff
@@ -0,0 +1,39 @@
+steps:
+  - label: ":terraform: :mag: validate"
+    plugins:
+      - docker#v5.9.0:
+          image: hashicorp/terraform:1.5
+          command: ["validate"]
```

Comment:
Personal preference here, I prefer the command to be included at the top level as opposed to the docker command.  Makes it a smidge easier to refactor and reads a little better IMO.